newIDE/app/src/ProjectManager/index.js

```diff
@@ -1069,6 +1069,7 @@ export default class ProjectManager extends React.Component<Props, State> {
               value={searchText}
               onRequestSearch={this._onRequestSearch}
               onChange={this._onSearchChange}
+              placeholder={t`Search manager`}
```

Comment:
I don't know if this is very clear. If we were really accurate, it would be "Search in layout, external events, external layouts and functions/behaviors names". Might not be worth adding a placeholder here

Reply:
+1, or "Search in project"

Reply:
Actually, I think a placeholder is necessary for this one, it was one of the few that I did not know what it searched for at first glance!
Florian suggested "Search in project" instead.

Reply:
Let's go with `Search in project` then! We'll see if users are really confused